### PR TITLE
docs: add two weekly report blog

### PR DESCRIPTION
**Pull request by KID-G**
Changes:

add two weekly reports blog （sync from WeChat Content:[weekly report 11.10](https://mp.weixin.qq.com/s/amRIuqMOR-_PETwlI10KDA) [weekly report 11.23](https://mp.weixin.qq.com/s/nIXDfhrm4ZyNyPlPV2aPlw))

### CODE REVIEW

File: blog/zh/blog/2022/11/10/weekly-report-1110.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+**问题描述**：为 `jwt-auth` 插件隐藏凭证，可避免在上游泄露 Token。
+
+### Issue #1416
+
+**链接**：https://github.com/apache/apisix-ingress-controller/issues/1416
+
+**问题描述**：在某些情况下，5000 端口可能会被其它进程占用，是否可以支持自定义端口。
+
+
```
Comment (SylviaBABY) [inline blog/zh/blog/2022/11/10/weekly-report-1110.md]: delete this line

File: blog/zh/blog/2022/11/10/weekly-report-1110.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+- [支持处理上游 header 时自定义 `X-Forwarded-Port` 参数](https://github.com/apache/apisix/pull/8266)（贡献者：[biakecw](https://github.com/biakecw)）
+
+- [修正 Dashboard etcd 连接中断时数据不同步问题](https://github.com/apache/apisix-dashboard/pull/2650)（贡献者：[bzp2010](https://github.com/bzp2010)）
+
+Apache APISIX 的项目官网和 Github 上的 Issue 上已经积累了比较丰富的文档教程和使用经验，如果您遇到问题可以翻阅文档，用关键词在 Issue 中搜索，也可以参与 Issue 上的讨论，提出自己的想法和实践经验。
+
+## 近期博文推荐
+
+
```
Comment (SylviaBABY) [inline blog/zh/blog/2022/11/10/weekly-report-1110.md]: ditto

File: blog/zh/blog/2022/11/23/weekly-report-1123.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+- [腾讯蓝鲸 API 网关如何借助 APISIX 实现产品升级与业务完善](https://apisix.apache.org/zh/blog/2022/11/05/tencent-blueking-with-apisix/)
+
+    本文从蓝鲸网关的迭代过程和应用 APISIX 后的产品实践角度，为大家呈现了基于 APISIX 的网关应用实例。
+
+- [从 Traefik 到 APISIX，汽车智能计算平台公司「地平线」在 Ingress Controller 的探索和实践](https://apisix.apache.org/zh/blog/2022/10/28/apisix-ingress-with-horizon-ai/)
+
+    地平线作为一家致力于汽车自动驾驶芯片的公司，在业务迭代中为何从 Traefik 切换到了 APISIX Ingress 去支撑业务？
+
+
```
Comment (SylviaBABY) [inline blog/zh/blog/2022/11/23/weekly-report-1123.md]: ditto

File: blog/zh/blog/2022/11/23/weekly-report-1123.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+**问题描述**：为避免主机匹配争议，将默认 HTTP 路由器从 `radixtree_uri` 更改为 `radixtree_host_uri`。
+
+### Issue #8311
+
+**链接**：https://github.com/apache/apisix/8311
+
+**问题描述**：希望 Kubernetes 的服务发现可以支持更多的配置项，用以应对不同的验证场景。
+
+
```
Comment (SylviaBABY) [inline blog/zh/blog/2022/11/23/weekly-report-1123.md]: ditto

**Comment by SylviaBABY** [comment]
you can click here to check the Lint error details
image

**Comment by KID-G** [comment]
> you can click here to check the Lint error details image

Thank you!

**Review by SylviaBABY — APPROVED** [review]
LGTM

**Review by SkyeYoung — APPROVED** [review]

**Pull request merged**